charts/kueue/templates/rbac/role.yaml

```diff
@@ -55,13 +55,6 @@ rules:
       - list
       - update
       - watch
-  - apiGroups:
```

Comment:
create a separate PR for this

Reply:
it was already in main, I just rebased it